### ISSUE TITLE
Sending multiple commands will fail

### ISSUE THREAD

**Issue by teejay-87** [anotherjulien/MyHOME#1]
If I try to shut an entire floor off (for example via Homekit, "Siri turn off first floor", _**but not limited to that**_) most commands are ignored and an error is shown in the HA log:

```
Dettagli registro (ERROR)
Logger: homeassistant.core
Source: custom_components/myhome/gateway.py:282
First occurred: 12:59:54 (3 occurrences)
Last logged: 12:59:55

Error executing service: <ServiceCall light.turn_off (c:62dc88041d0d75af0487115f3106e2ae): entity_id=['light.scala_primo_piano']>
Error executing service: <ServiceCall light.turn_off (c:278c92ae0608a38cc35cf82635e8adb5): entity_id=['light.bagno_primo_piano']>
Error executing service: <ServiceCall light.turn_off (c:a4a488b58dc37a79b8ed335b3f466c2a): entity_id=['light.studio']>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/core.py", line 1467, in catch_exceptions
    await coro_or_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1486, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 499, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 664, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 536, in _handle_entity_call
    await result
  File "/config/custom_components/myhome/light.py", line 189, in async_turn_off
    await self._gateway.send(OWNLightingCommand.switch_off(self._where))
  File "/config/custom_components/myhome/gateway.py", line 282, in send
    await command_session.send(message=message)
  File "/usr/local/lib/python3.8/site-packages/OWNd/connection.py", line 482, in send
    negotiation_result = await self._negotiate()
  File "/usr/local/lib/python3.8/site-packages/OWNd/connection.py", line 207, in _negotiate
    raw_response = await self._stream_reader.readuntil(OWNSession.SEPARATOR)
  File "/usr/local/lib/python3.8/asyncio/streams.py", line 629, in readuntil
    raise exceptions.IncompleteReadError(chunk, None)
asyncio.exceptions.IncompleteReadError: 0 bytes read on a total of None expected bytes
```

AFAIK, the SCS bus only accept a limited number of command per period of time. For example, if you physically press two light switches at the same time, most of the time just one of the two get turned on.

I think this problem should be solved by artificially limiting the number of inputs sent in parallel on the bus, and sending them in sequence with a minimum delay one from each other.

**Comment by anotherjulien** [comment]
Can you remind me what gateway you are using?
The time for HA to process messages for each individual light should be long enough to allow for the gateway to receive it.
At least I can reliably turn on or off all 34 lights I have with none missing.

**Comment by teejay-87** [comment]
I'm using the MH201 that is indeed not a gateway.
But I think the problem is a limit of the SCS bus, the same thing happens if I physically press two wall-switches at once, often only one light get turned on.

It's strange you have no problem with so many lights, I have with 10 or less.

**Comment by anotherjulien** [comment]
There are indeed some sorts of rate limits on the electrical SCS bus itself, but here the issue is in the communication with the gateway.
The gateway is apparently able to buffer messages to prevent collisions on the bus, so itself cannot generate events that you manually can by pressing two buttons at once.

I think the problem here is that the gateway cannot handle too many sessions at once.
Someone else reported similar errors on an MH201, so I think its specs are indeed not those of a full fledged gateway.

I'll be working on a "retry" mechanism rather than a built-in rate limiter.
These are changes that will need to happen in OWNd, I'll publish a new version as soon as possible.

**Comment by teejay-87** [comment]
Ok, that's fair.
Let me know if I can help any way!

**Comment by anotherjulien** [comment]
I've published OWNd v0.7.13
Can you try to change the requirement in `manifest.json` to fit this version, restart HA and give it a go?

**Comment by teejay-87** [comment]
Yep! It works!
